src/main/resources/swagger/thurloe.yaml

```diff
@@ -324,11 +324,13 @@ paths:
         schema:
           {
             "type": "object",
-            "required": ["userId", "notificationId"],
```

Comment:
The `Notification` case class looks like it still expects to always have a `notificationId`. Should it still be required?